bfrec

```diff
@@ -146,6 +146,53 @@ fi
 bfb_location=/lib/firmware/mellanox/boot
 capsule_location=/lib/firmware/mellanox/boot/capsule
 
+parse_and_set_capsule_versions()
+{
+    local capsule_file="$1"
+
+    if [ -z "$capsule_file" ]; then
+        echo "ERROR: No capsule file specified"
+        exit 2
+    fi
+
+    if [ ! -f "$capsule_file" ]; then
+        echo "ERROR: Capsule file '$capsule_file' not found"
+        exit 2
+    fi
+
+    # Use bfver to parse the ATF and UEFI version from the capsule file.
+    BFVER_OUTPUT=$(bfver -f "$capsule_file")
```

Comment:
`bfver` is a dependency. I suggest we check whether the command is found before it is called.